requirements.txt

```diff
@@ -4,4 +4,5 @@ pandas==0.19.2
 Flask==0.12.1
 nltk==3.2.2
 py4j==0.10.4
--e git+https://github.com/castorini/Castor.git#egg=sm-cnn-1.0.0
+#-e git+https://github.com/castorini/Castor.git#egg=sm-cnn-1.0.0
+-e git+https://github.com/gauravbaruah/Castor.git@corpus-idf#egg=sm_cnn-1.0.0
```

Comment:
Uncomment line 7 and delete line 8 because once the PR is merged we want the latest `sm_cnn` module to be same as the one found at https://github.com/castorini/Castor.git.